boards/arm/nrf52_adafruit_feather/nrf52_adafruit_feather.dts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2018 LEDCity AG
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832.dtsi>
+
+/ {
+	model = "nRF52 Adafruit Feather";
+	compatible = "adafruit,nrf52_adafruit_feather", "nordic,nrf52832-qfaa",
+		     "nordic,nrf52832";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "ok";
+};
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * http://docs.zephyrproject.org/devices/dts/flash_partitions.html
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x8000>;
+		};
+		slot0_partition: partition@8000 {
+			label = "image-0";
+			reg = <0x00008000 0x34000>;
+		};
+		slot1_partition: partition@3c000 {
+			label = "image-1";
+			reg = <0x0003c000 0x34000>;
+		};
+		scratch_partition: partition@70000 {
+			label = "image-scratch";
+			reg = <0x00070000 0xD000>;
+		};
+
+		/*
```

Comment:
Can you add partition for storage as wll, you can take it from [nrf52_pca10040](https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf52_pca10040/nrf52_pca10040.dts#L63-L68).